python/cudf/cudf/_libxx/table.pyx

```diff
@@ -227,6 +230,27 @@ cdef class Table:
             self._index._data.columns
         )
 
+    @staticmethod
+    cdef columns_from_ptr(
```

Comment:
Sounds like this is better off being a `staticmethod` of `TableColumns`. Something like `TableColumns.from_unique_ptr`?

Reply:
oops - I forgot to actually remove `TableColumns`. I was using it before I realized one could just return the list itself without having a whole object for this.

Reply:
Should be gone now.

Reply:
Cool - I still find it a bit awkward to have this function attached to `Table`, as it really has little to do with `Table`. I'd just make a standalone function along the lines of:

```
cdef columns_from_cudf_table(unique_ptr[table] ptr):
    pass
```